docs/runbooks/CDIDataImportCronOutdated.md

```diff
@@ -23,61 +27,51 @@ VMs might fail to start because no boot source is available for cloning.
 
 ## Diagnosis
 
-1. Check the cluster for a default storage class:
+1. Check the cluster for a default (Kubernetes and/or virtualization) storage class:
 
    ```bash
-   $ kubectl get sc
+   $ kubectl get sc -o jsonpath='{.items[?(.metadata.annotations.storageclass\.kubernetes\.io\/is-default-class=="true")].metadata.name}'
+
+   $ kubectl get sc -o jsonpath='{.items[?(.metadata.annotations.storageclass\.kubevirt\.io\/is-default-virt-class=="true")].metadata.name}'
    ```
 
-   The output displays the storage classes with `(default)` beside the name of
-   the default storage class. You must set a default storage class, either on
-   the cluster or in the `DataImportCron` specification, in order for the
-   `DataImportCron` to poll and import golden images. If no storage class is
-   defined, the DataVolume controller fails to create PVCs and the following
-   event is displayed: `DataVolume.storage spec is missing accessMode and no
-   storageClass to choose profile`.
+   The output displays the default (Kubernetes and/or virtualization) storage
```

Comment:
earlier you used `(Kubernetes or KubeVirt)` why the difference here?

Reply:
Aligned

Reply:
can you just clarify why you opted to use virtualization instead of KubeVirt?

Reply:
Yes, I saw it in the previous version. But do you know if there is a reason behind it, or from the CDI point of view both terms are technically correct?

Reply:
[in this runbook](https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/CDIDefaultStorageClassDegraded.md?plain=1) for example, we can see the usage of `KubeVirt storage class` and `Kubernetes or KubeVirt` which I think makes more sense

Reply:
You are right, missed that. I don't see any technical reason, but let me check and decide on consistent phrase.

Reply:
Discussing it with @akalenyu and @ousleyp , we decided to align refs to `default virtualization storage class`.